test/runtests.jl

```diff
@@ -1,18 +1,19 @@
 using FMIndexes
 using Combinatorics
-using Base.Test
+using Test
+using Random
 
-srand(12345)
+Random.seed!(12345)
 
 # simple DNA sequence type
 @enum Nuc A C G T
-type DNASeq
+mutable struct DNASeq
```

Comment:
Looks like this used to be immutable - does it need to be mutable now? Note: if the `data` field is a vector, you can still mutate the underlying vector even if the `struct` wrapping it is technically immutable.

Reply:
In old julia syntax, 'type' is mutable, and 'immutable' is immutable, so I just maintained the original mutability.  But I think you're right that it doesn't need to be mutable, but it's just runtests.jl so i'm not sure this matters either way.

Reply:
Oh right - I thought `type` was the old `struct`, I forgot that `type` was mutable by default.

>but it's just runtests.jl so i'm not sure this matters either way.

This is true. I don't feel strongly about changing it 🤷‍